pkg/sdk/errors.go

```diff
@@ -45,3 +45,174 @@ func decodeDriverError(err error) error {
 
 	return err
 }
+
+const ghIssueBodyTemplate = `
```

Comment:
template should be changed to be more matching with the current one, e.g. fields like Provider version / TF version should also be here